### PR TITLE
Fixes bluespace bread (and other slice foods as well)

### PR DESCRIPTION
**Pull request by DarkPyrolord**
From: #4330
As it says in the title, a quick little fix for slice foods, infinite storage inside of a loaf of bread should no longer be possible, things such as this below.

thebread1
thebread2

🆑 DarkPyrolord
fix: Sliceable food items are no longer hammerspace capable 
/🆑

### CODE REVIEW

**Comment by KingPhilipIII** [comment]
Damn. Hammerspace inside of bread sounds hilarious.

**Comment by FlattestGuitar** [comment]
Not a fix.

Heavy balance change.

Heads, please block this, this makes the game unplayable.

**Comment by Spartan6** [comment]
*scream

**Comment by taukausanake** [comment]
I thought this was fixed with the mentioned PR. Wow. Still want my bluespace cake and now bluespace bread. We need to make that happen

**Comment by Fox-McCloud** [comment]
Goodness, the code for stuffing things in food is awful. Who wrote this?

`user.client.screen -= W`

File: code/modules/food_and_drinks/food/snacks.dm [hunk 1]
```diff
@@ -130,14 +132,19 @@
 			istype(W, /obj/item/weapon/hatchet) \
 		)
 		inaccurate = 1
-	else if(W.w_class <= WEIGHT_CLASS_SMALL && istype(src,/obj/item/weapon/reagent_containers/food/snacks/sliceable))
+	if(total_w_class > MAX_WEIGHT_CLASS)
```
Comment (Fox-McCloud) [inline code/modules/food_and_drinks/food/snacks.dm]: I don't think we want to be able to stuff things in nonsliceable food.

Comment (DarkPyrolord) [reply code/modules/food_and_drinks/food/snacks.dm]: I blame my lack of sleep, one moment

Comment (tigercat2000) [reply code/modules/food_and_drinks/food/snacks.dm]: @DarkPyrolord You replaced the wrong `if()` statement :stuck_out_tongue: 
Leave this one as it was, 
`else if(W.w_class <= WEIGHT_CLASS_SMALL && istype(src,/obj/item/weapon/reagent_containers/food/snacks/sliceable))`

File: code/modules/food_and_drinks/food/snacks.dm [hunk 2]
```diff
@@ -130,10 +132,10 @@
...
 		if(contents.len)
```
Comment (tigercat2000) [inline code/modules/food_and_drinks/food/snacks.dm]: And change this one to `if(total_w_class > MAX_WEIGHT_CLASS)`

File: code/modules/food_and_drinks/food/snacks.dm [hunk 3]
```diff
@@ -130,14 +132,19 @@
 			istype(W, /obj/item/weapon/hatchet) \
 		)
 		inaccurate = 1
-	else if(W.w_class <= WEIGHT_CLASS_SMALL && istype(src,/obj/item/weapon/reagent_containers/food/snacks/sliceable))
+	if(total_w_class > MAX_WEIGHT_CLASS && istype(src,/obj/item/weapon/reagent_containers/food/snacks/sliceable))
```
Comment (tigercat2000) [inline code/modules/food_and_drinks/food/snacks.dm]: You replaced the wrong if() statement; Leave this one as `else if(W.w_class <= WEIGHT_CLASS_SMALL && istype(src,/obj/item/weapon/reagent_containers/food/snacks/sliceable))`

File: code/modules/food_and_drinks/food/snacks.dm [hunk 4]
```diff
@@ -130,14 +132,19 @@
...
+		if(contents.len)
```
Comment (tigercat2000) [inline code/modules/food_and_drinks/food/snacks.dm]: And change this one to `if(total_w_class > MAX_WEIGHT_CLASS)`

**Pull request merged**